index.html

```diff
@@ -61,5 +60,141 @@ <h1 class="section-title">I’m Polina Glad to see you!</h1>
+          sign-ups required.
+        </p>
+        <div class="butonns-languages">
+          <p class="btn-lg">html</p>
+          <p class="btn-lg">css</p>
+          <p class="btn-lg">javaScritp</p>
+        </div>
+        <button type="button" class="btn-project">See Project</button>
+      </div>
+      <div class="card">
+        <img src="images/image3.png" alt="Yoga" />
+        <h1 class="card-title">TONIC</h1>
+        <p class="card-description">
+          CANOPY <span> &#9679; Back End Dev &#9679; 2015</span>
+        </p>
+        <p class="card-text-content">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <div class="butonns-languages">
+          <p class="btn-lg">html</p>
+          <p class="btn-lg">css</p>
+          <p class="btn-lg">javaScritp</p>
+        </div>
+        <button type="button" class="btn-project">See Project</button>
+      </div>
+      <div class="card">
+        <img src="images/image4.png" alt="Proffesional art printing" />
+        <h1 class="card-title">TONIC</h1>
+        <p class="card-description">
+          CANOPY <span> &#9679; Back End Dev &#9679; 2015</span>
+        </p>
+        <p class="card-text-content">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <div class="butonns-languages">
+          <p class="btn-lg">html</p>
+          <p class="btn-lg">css</p>
+          <p class="btn-lg">javaScritp</p>
+        </div>
+        <button type="button" class="btn-project">See Project</button>
+      </div>
+    </section>
+    <section class="about-myself">
+      <div class="about-myself-content">
+        <h1 class="section-title">About Myself</h1>
+        <p>
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <p><a href="#" class="main-text">LET’S CONNECT</a></p>
+        <ul>
+          <li>
+            <a href="#"
+              ><img src="images/icon-twitter.png" alt="Icon-Twitter"
+            /></a>
+          </li>
+          <li>
+            <a href="#"
+              ><img src="images/icon-linkedin.png" alt="Icon-Linkedin"
+            /></a>
+          </li>
+          <li>
+            <a href="#"><img src="images/icon-M.png" alt="Icon-M" /></a>
+          </li>
+          <li>
+            <a href="#"
+              ><img src="images/icon-github.png" alt="Icon-GitHub"
+            /></a>
+          </li>
+          <li>
+            <a href="#"
+              ><img src="images/icon-fingers.png" alt="Icon-Fingers"
+            /></a>
+          </li>
+        </ul>
+        <button type="button" class="btn-project">Get my resume</button>
+        <div class="skills-container">
```

Comment:
- [ ] I see that you have grouped the list of project technologies (e.g., "JavasScript", "HTML" and "CSS") under `Languages` skill using `ul` -`li`, good job there! However, the skills(languages, Frameworks, Skills) themselves should be grouped using `ul` - `li` as well.  This is what the previous reviewer meant by grouping all the elements in the Skills section.